JPS_Ontology/ontology/ontoems/OntoEMS.owl

```diff
@@ -640,7 +640,7 @@
 
 
     <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontoems/AirPollutantConcentration">
-        <rdfs:subClassOf rdf:resource="http://theworldavatar.com/resource/ontouom/MassOfSubstanceConcentration"/>
+        <rdfs:subClassOf rdf:resource="http://theworldavatar.com/resource/ontouom/Concentration"/>
```

Comment:
In the latest OntoUOM extension, Yi Kai followed the new naming convention for new ontouom concepts, e.g. "https://www.theworldavatar.com/kg/ontouom/MassOfSubstanceConcentration" (https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontouom/MassOfSubstanceConcentration)
However we do it, it will be inconsistent, but maybe we should continue with this new approach (at least for new concepts) even though that means that the IRIs will not resolve as URLs. @sm453 What is your perspective on that?

Reply:
Absolutely agree.